proto/call.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2014 The Cockroach Authors.
```

Comment:
This file was moved from `client/call.go` to `proto/call.go` with some edits to adjust for the package movement and adding the `Call` suffix to the methods. Github is showing it as brand new code, but it isn't.